products/rhel9/profiles/default.profile

```diff
@@ -574,3 +574,4 @@ selections:
     - file_permissions_etc_audit_rulesd
     - configure_openssl_tls_crypto_policy
     - configure_openssl_crypto_policy
+    - audit_rules_login_events
```

Comment:
Please add also the other removed rules (audit_rules_usergroup_modification and audit_rules_unsuccessful_file_modification) to prevent their removal from RHEL 9 data stream.